### PR TITLE
updating-gitignore files

### PR DESCRIPTION
**Pull request by ManjulaPaluru**
updated the git ignore with intellij generated files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,6 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+*.iml
```
Comment (ssaguturu) [inline .gitignore:25]: Can you add some comments, like what kind of files are these?

**Review by ssaguturu — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ManjulaPaluru) [reply .gitignore:25]: sure, i have added the iml and .idea files

Comment (ManjulaPaluru) [reply .gitignore:25]: i have added details in the PR description.

Comment (ssaguturu) [reply .gitignore:25]: Thanks, I would have seen that... my bad!

**Review by ssaguturu — APPROVED** [review]

**Pull request merged**